src/java.base/share/classes/sun/nio/ch/SocketInputStream.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2022,2023 Oracle and/or its affiliates. All rights reserved.
```

Comment:
I think the Oracle copyright header checker will ask you to put a space after the comma.